cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -226,41 +230,43 @@ public void testBrokerCapacities() throws JsonProcessingException {
         assertThat(getCapacityConfigurationFromEnvVar(resource, ENV_VAR_CRUISE_CONTROL_CAPACITY_CONFIGURATION), is(capacity.toString()));
 
         // Test generated disk capacity
-        JbodStorage jbodStorage = new JbodStorage();
-        List<SingleVolumeStorage> volumes = new ArrayList<>();
-
-        PersistentClaimStorage p1 = new PersistentClaimStorage();
-        p1.setId(0);
-        p1.setSize("50Gi");
-        volumes.add(p1);
+        JbodStorage jbodStorage = new JbodStorageBuilder()
+                .withVolumes(
+                        new PersistentClaimStorageBuilder().withDeleteClaim(true).withId(0).withSize("50Gi").build(),
+                        new PersistentClaimStorageBuilder().withDeleteClaim(true).withId(1).build()
+                ).build();
 
-        PersistentClaimStorage p2 = new PersistentClaimStorage();
-        p2.setId(1);
-        volumes.add(p2);
+        Map<String, Quantity> requests = new HashMap<>(1);
+        requests.put(Capacity.RESOURCE_TYPE, new Quantity("400m"));
```

Comment:
We could use `Collections.singletonMap()` here and a few places below

Reply:
I think that in a new code we tend to use `Map.of()`.